data/intro missions.txt

```diff
@@ -451,3 +451,143 @@ conversation "end of intro missions"
 	label next
 	`	There's an awkward pause. You're unsure of how to respond. Then he says, "Thing is, this here planet is one of the most exclusive retirement communities in the galaxy. Plenty of the locals are probably worth more than a billion credits. But they're mostly financial or business types, or old money. I'm rich enough to be here, but I just hope they're willing to socialize with someone who made his fortune flying a ship."`
 	`	He hands you a credit chip worth <payment>, and says, "Best of luck to you, Captain. Don't hesitate to look me up if you're ever in the area."`
+
+
+
+mission "Intro: Follow Up - Hai Before Main Plot"
+	name "Bring James to <planet>"
+	description "Take James to <destination> to meet the Hai."
+	blocked `You consider paying a visit to James to tell him about the Hai, but decide to wait until you have a free seat available on your ship. Just in case.`
```

Comment:
This mission is the beginning of a string, and therefore shouldn't have a blocked message. Blocked dialogs are only for situations where you're continuing a mission string.

Reply:
done, though I thought it was nice to provide a hint that a potential mission was there.  also removed the other similar blocked message for Hai After Main Plot.